.travis/install_phpstan.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+set -ev
+
+composer update --prefer-dist --no-interaction --prefer-stable ${COMPOSER_FLAGS}
```

Comment:
Why is this script necessary? Aren't we already using `composer update` elsewhere?

Reply:
We don't update the project resources if we just run the lint (or any other non-test) command.

Maybe I can improve this command, to just run composer update for the phpstan `composer.json` file